www/js/incident/post-trip-prompt.js

```diff
@@ -224,39 +159,46 @@ angular.module('emission.incident.posttrip.prompt', ['emission.plugin.logger'])
     });
     $window.cordova.plugins.notification.local.on('trigger', function (notification, state, data) {
         // alert("triggered, no action");
-        Logger.log("triggered, no action");
-        if (!$ionicPlatform.is('ios')) {
-            Logger.log("notification is displayed even when app is in foreground, ignoring trigger");
-            return;
-        }
+        Logger.log("Notification triggered");
         if (!checkCategory(notification)) {
-            Logger.log("notification "+notification+" is not an incident report, returning...");
+            Logger.log("notification "+notification+" is not an mode choice, returning...");
             return;
         }
         cleanDataIfNecessary(notification, state, data);
-        promptReport(notification, state, data).then(function(res) {
-          if (res == true) {
-            Logger.log("About to go to incident map page");
-            displayCompletedTrip(notification, state, data);
-          } else {
-            Logger.log("Skipped incident reporting");
-          }
-        });
+        if($ionicPlatform.is('ios')) {
+          promptReport(notification, state, data).then(function(res) {
+            if(res == true) {
+              Logger.log("About to go to prompt page");
+              displayCompletedTrip(notification, state, data);
+            } else {
+              Logger.log("Skipped incident reporting");
+            }
+          });
+        } else {
+          Logger.log("About to go to prompt page");
+          displayCompletedTrip(notification, state, data);
+        }
     });
     $window.cordova.plugins.notification.local.on('click', function (notification, state, data) {
       // alert("clicked, no action");
+      Logger.log("Notification, click event");
       if (!checkCategory(notification)) {
-          Logger.log("notification "+notification+" is not an incident report, returning...");
+          Logger.log("notification "+notification+" is not an mode choice, returning...");
           return;
       }
       cleanDataIfNecessary(notification, state, data);
       displayCompletedTrip(notification, state, data);
     });
-  }
+  };
 
+  /*
+   * Unregister the post-trip notification for cci-berkeley
+   * because they want to make it opt-in. Since the base app registers by
+   * default, we need to explicitly unregister, not just comment out.
+   */
   $ionicPlatform.ready().then(function() {
-    ptap.registerTripEnd();
-    ptap.registerUserResponse();
+    ptap.unregisterTripEnd();
```

Comment:
this is why you don't want to merge blindly :) Read the docs!

Reply:
I will also revert this one back to what it was. Thank you.